packages/suite/src/components/wallet/CoinjoinSummary/CoinjoinStatus.tsx

```diff
@@ -138,25 +155,30 @@ interface CoinjoinStatusProps {
 export const CoinjoinStatus = ({ session, accountKey }: CoinjoinStatusProps) => {
     const [isLoading, setIsLoading] = useState(false);
     const [isWheelHovered, setIsWheelHovered] = useState(false);
+    const { isTorEnabled } = useSelector(selectTorState);
 
     const menuRef = useRef<HTMLUListElement & { close: () => void }>(null);
     const theme = useTheme();
     const dispatch = useDispatch();
 
-    const { maxRounds, signedRounds, paused, phaseDeadline, sessionDeadline, phase } = session;
+    const { maxRounds, signedRounds, paused, phaseDeadline, sessionDeadline, phase, interrupted } =
+        session;
 
     const progress = calculateSessionProgress(signedRounds, maxRounds);
     const isPaused = !!paused;
 
+    const isResumeDisable = !isTorEnabled && (isPaused || !!interrupted);
```

Comment:
Perhaps we should not even rely on `isTorEnabled`? `interrupted` could be changed to something like `isBlocked`, so we render the paused state based on `isPaused` and then for whatever reason disable resuming with `isBlocked`. It would also simplify addition of [other reasons](https://github.com/trezor/trezor-suite/issues/7063) for resuming deactivation. :) What do you think?

Reply:
I think that `isTorEnabled` is totally necessary here, since we want to allow the user to resume the process if it was paused and Tor is enable.

Reply:
Is the `interrupted` flag even required in this case?

Reply:
Yes, it is used when Tor is enable, if the CoinJoin process was paused because of an interruption it gets automatically resumed, otherwise if it was paused by the user it stays paused.

Reply:
Up to you but I think `isResumeDisable` could be a selector! :)

Reply:
That sounds nice idea, but I suggest to do it in the next issue with the rest of the resume blocker reasons https://github.com/trezor/trezor-suite/issues/6908 :)

Reply:
@dahaca you should be happy I ended up creating a selector for this https://github.com/trezor/trezor-suite/pull/6796/files#diff-086622f10df304f5ceeda6cdf02c5ca9c6cd2ddcb337b5e1c15fb2552381fd25R373

I had to create it based on the reason that it is blocking it so we can then use it for text in tooltips an so on. And then create other selectors for the other reasons, or similar.